Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -407,7 +407,7 @@ - (void)ensureViewState:(YapDatabaseReadTransaction *)transaction
     if (self.messageCellType == OWSMessageCellType_Unknown) {
         // Messages of unknown type (including messages with missing attachments)
         // are rendered like empty text messages, but without any interactivity.
-        DDLogWarn(@"%@ Treating unknown message as empty text message: %@", self.logTag, message.description);
+        DDLogWarn(@"%@ Treating unknown message as empty text message: %@ %llu", self.logTag, message.class, message.timestamp);
```

Comment:
test failing because `description` uses a transaction, but we shouldn't be logging message.description anyway.